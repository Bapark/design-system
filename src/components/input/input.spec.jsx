import React from 'react';
import { shallow } from 'enzyme';
import Input from './input';

function render(props) {
  return shallow(<Input {...props} id="hi" />);
}

test('rendering', () => {
  const wrapper = render();
  expect(wrapper.find('input').prop('id')).toEqual('hi');
});

test('passing props through', () => {
  const wrapper = render({ className: 'hello', required: true });
  expect(wrapper.find('input').prop('className')).toEqual('hello');
  expect(wrapper.find('input').prop('required')).toEqual(true);
});
