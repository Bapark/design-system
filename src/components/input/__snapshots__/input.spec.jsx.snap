// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`passing props through 1`] = `
<input
  className="hi"
  id="hi"
  required={true}
/>
`;

exports[`rendering 1`] = `
<input
  className="input"
  id="hi"
/>
`;
